### PR TITLE
Feature/mobile config options and fixes

### PR DESCRIPTION
**Pull request by builtbywill**

### CODE REVIEW

**Review by caseyveronica4 — APPROVED** [review]
seems good :thumps:

File: src/editor/editor-mathfield.js [hunk 1]
```diff
@@ -204,7 +204,7 @@ function MathField(element, config) {
 
     // Only display the virtual keyboard toggle if the virtual keyboard mode is
     // 'manual'
-    if (this.config.virtualKeyboardMode === 'manual') {
+    if (this.config.virtualKeyboardMode === 'manual' || this.config.showVirtualKeyboardButton) {
```
Comment (lily514) [inline src/editor/editor-mathfield.js:207]: @builtbywill so these configs already existed?

Comment (builtbywill) [reply src/editor/editor-mathfield.js:207]: @lily514 `isNativeMobileKeyboardEnabled` and `showVirtualKeyboardButton` are just a new properties I am checking for

**Review by lily514 — APPROVED** [review]
I officially have no idea what this is doing, but sure. :thumps:

**Pull request merged**